lib/companies/companies.ex

```diff
@@ -34,8 +34,7 @@ defmodule Companies.Companies do
   id next.
   """
   def recent do
-    %{order: [desc: :inserted_at, desc: :id]}
-    |> all()
+    all(%{order: [desc: :inserted_at, desc: :id]})
```

Comment:
good call on removing the pipe when piping with a single function. 

Would it make sense to add Credo to this project to enforce such rules across the codebase.

Reply:
I was instructed  to do so by our beloved @doomspork  :)

Reply:
Hahaha! You're too funny @gemantzu 

@maartenvanvliet I'm open to using Credo. I find it very helpful. I'll leave it up to someone else to add to the project. I believe we use it on some other projects so we may be able to borrow the config.